src/main/java/no/nav/tag/finnkandidatapi/kandidat/KandidatRepository.java

```diff
@@ -58,6 +58,24 @@ public KandidatRepository(JdbcTemplate jdbcTemplate) {
         }
     }
 
+    public List<Kandidat> hentKandidater() {
+        String query =
+                "SELECT k.* " +
+                "FROM kandidat k " +
+                        "INNER JOIN " +
```

Comment:
Marerittminner fra utvekslingsoppholdet på San Jose State University der Dr. Lee drar opp én og én student til tavla for å lage queries for "kompliserte spørringer" på sparket. Grøss.